module/scraper/scraper.py

```diff
@@ -0,0 +1,84 @@
+""""scraper module"""
+from datetime import datetime as time
+import locale
+from bs4 import BeautifulSoup
+import requests
+import yaml
+from yaml.loader import SafeLoader
+
+with open('config/settings.yaml', 'r', encoding="UTF-8") as file:
+    data = yaml.load(file, Loader=SafeLoader)
+
+def get_html(url):
+    response = requests.get(url, timeout=10)
+    return response.text
+
+def find_info(articles_list: list, article_number: int):
+    #finds all the info of an article
+    article = articles_list[article_number]
+    href_article = article.find_all('a',href=True, title=True)
+    title_article = href_article[1].get_text()
+    div_article = article.find('div',{'class' : 'slide-meta' })
+    link_article = article.find('a')
+    link_article = link_article['href']
+    if div_article is not None:
+        time_article = div_article.find('time').get_text()
+    else:
+        time_article = None
+    #find the tag of the article and content
+    html_article = get_html(link_article)
+    soup_article = BeautifulSoup(html_article, 'html.parser')
+    tag_article = soup_article.find('a', {'rel' : 'tag' })
+
+    if tag_article is None:
+        tag_article = soup_article.find('a', {'rel' : 'v:url' }, {'property' : 'v:title'}, href=True).get_text()
+    else:
+        tag_article = soup_article.find('a', {'rel' : 'tag' }).get_text()
+
+    content_article = soup_article.find('div', {'class' : 'entry-content'})
+    if content_article is not None:
+        content_article = content_article.find('p')
+    else:
+        content_article = None
```

Comment:
"if content_article is not None assign a value, otherwise (else, so if it is None) assign None", but I think that in the `else` case it already is None right? so the else is not necessary